locales/en/LC_MESSAGES/volto.po

```diff
@@ -3524,53 +3524,53 @@ msgstr "Services"
 #: components/ItaliaTheme/View/ServizioView/ServizioAccedi
 # defaultMessage: Accedi al servizio
 msgid "servizio_canale_digitale"
-msgstr ""
+msgstr "Access the service"
 
 #: components/ItaliaTheme/View/Commons/PageHeader/PageHeaderLinkServizio
 #: components/ItaliaTheme/View/ServizioView/ServizioAccedi
 # defaultMessage: Accedi al servizio
 msgid "servizio_canale_digitale_link"
-msgstr ""
+msgstr "Access the service"
 
 #: components/ItaliaTheme/View/ServizioView/ServizioComeFare
 # defaultMessage: Come fare
 msgid "servizio_come_fare"
-msgstr ""
+msgstr "How to"
```

Comment:
"How to" molto molto molto generico. Andrei con un "Instructions" che è più dritto al punto :)